### PR TITLE
Strings and changelog

### PR DESCRIPTION
**Pull request by LWinterberg**
Updates changelog

### CODE REVIEW

File: src/cloud/audiocom/ShareAudioDialog.cpp [hunk 1]
```diff
@@ -652,7 +652,8 @@ void ShareAudioDialog::InitialStatePanel::PopulateInitialStatePanel(
             anonInfoPanel = s.StartInvisiblePanel();
             {
                AccessibleLinksFormatter privacyPolicy(XO(
-                  "Your audio will be uploaded to our sharing service: %s,%%which requires a free account to use."));
+                  /*i18n-hint: %s substitutes for audio.com. %% creates a linebreak in this context. */
+                  "Sharing audio requires a free %s account linked to Audacity. %%Press \"Link account\" above to proceed."));
```
Comment (LWinterberg) [inline src/cloud/audiocom/ShareAudioDialog.cpp:656]: The %% here creates a linebreak and is somehow tied to some accessibility object. I placed it where the resulting linebreak works well, and accessibility seems to work still as well.

File: CHANGELOG.txt [hunk 2]
```diff
@@ -1,5 +1,42 @@
...
+   * New exporter
+     Audacity now features a new export window with easier access to options such as sample rate and custom mapping (for 5.1 or 7.1 audio). Additionally, the "Browse" button uses the native file browser now!
+
+ Other changes
+   * For MP3, Audacity now always uses Joint Stereo mode, which always delivers the best possible quality.
+   * #4720 Simplified pasting logic.
+   * Simplified stereo tracks. Left and right channels now always have synchronized clip starts and ends, and the same sample rate in both channels. 
+   * #5014 Added cursors trimming and stretching left and right, and changed the I-beam cursor to be less similar to either.
+   * #4448 When importing audio, the project sample rate no longer changes.
+   * #5204 The spectrogram colors are now perceptually uniform, and the colormap has gotten a name: Roseus. It also can be used for dataviz outside of Audacity, see https://github.com/dofuuz/roseus for more info!
```
Comment (Paul-Licameli) [inline CHANGELOG.txt:21]: Do we want a thank-you mention of "dofuuz"?  (Myungchul Keum, already credited in the "About Audacity" dialog)

File: CHANGELOG.txt [hunk 3]
```diff
@@ -1,5 +1,42 @@
...
+   * New defaults: The Time Signature toolbar now gets shown, the solo button is set to multi-track mode, time tracks have a wider starting range.
+   * #5291 Unsquished the Audacity logo found in some places of the app.
+   * #2330 Clicking on the border of two clips no longer merges them.
+
+ Libraries
+   * lib-time-and-pitch implements a time stretching algorithm originating in Staffpad.
+   * Audacity now uses Conan 2.
+   * Continued extracting code into libraries and modules.
```
Comment (Paul-Licameli) [inline CHANGELOG.txt:29]: Especially for import/export.  And some other reorganizations.

**Comment by Paul-Licameli** [comment]
I found no grammar or spelling typos

File: CHANGELOG.txt [hunk 4]
```diff
@@ -18,15 +18,15 @@ Audacity 3.4 adds a musical view, time stretching and a new exporter.
    * Simplified stereo tracks. Left and right channels now always have synchronized clip starts and ends, and the same sample rate in both channels. 
    * #5014 Added cursors trimming and stretching left and right, and changed the I-beam cursor to be less similar to either.
    * #4448 When importing audio, the project sample rate no longer changes.
-   * #5204 The spectrogram colors are now perceptually uniform, and the colormap has gotten a name: Roseus. It also can be used for dataviz outside of Audacity, see https://github.com/dofuuz/roseus for more info!
+   * #5204 The spectrogram colors are now perceptually uniform, and the colormap has gotten a name: Roseus. It also can be used for dataviz outside of Audacity, see https://github.com/dofuuz/roseus for more info! (Thanks, Myungchul Keum)
```
Comment (Paul-Licameli) [inline CHANGELOG.txt:21]: @dofuuz thanks again!

**Pull request merged**